src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -58,4 +58,36 @@ public function iShouldSeeCountShipmentsInList(int $count): void
     {
         Assert::same($count, $this->indexPage->countItems());
     }
+
+    /**
+     * @Given I choose :shipmentState as a shipment state
```

Comment:
```suggestion
     * @When I choose :shipmentState as a shipment state
```

Also, can you move this declaration before the first function with `Then`?